tsconfig.json

```diff
@@ -1,15 +1,11 @@
 {
   "compilerOptions": {
-    "lib": ["es2019", "dom"],
-    "target": "es2017",
+    "target": "esnext",
```

Comment:
This needed to change to match the VS Code environment. 3rd party libs weren't working correctly.